turftopic/models/decomp.py

```diff
@@ -47,6 +47,7 @@ def __init__(
         ] = "sentence-transformers/all-MiniLM-L6-v2",
         vectorizer: Optional[CountVectorizer] = None,
         objective: Literal["orthogonality", "independence"] = "independence",
+        subset: str = None
```

Comment:
I'm not sure whether we should have this in this PR. (actually pretty sure we shouldn't). I'm not certain in the first place whether this is functionality we want. I'd prefer the negative/positive solution, and I would not want this to confuse our users while we're testing whether it makes sense.